cvat/apps/engine/static/engine/js/shapes.js

```diff
@@ -1629,6 +1625,53 @@ class RaysController extends PolyShapeController {
     }
 
 
+    createVanishingPoint(lineElement, isBigRotation, lineElements) {
+        let [a, b] = this.extractPoints(lineElement);
+        if (!isBigRotation) {
+            [a, b] = [b, a];
+        }
+        const newVanishingPoint = this.getPointFromRatio(a, b, 2);
```

Comment:
why 2?

Reply:
There can be any number here. I just have to put the vanishing point somewhere on the line, I don't see any meaningful place for it, so I've decided to put it at the same distance from the edge point as the distance between edge points (so ratio is 2 in this case).